### PR TITLE
Border radius as per material design specification

### PR DESCRIPTION
**Pull request by mungpara**

### CODE REVIEW

**Comment by macjohnny** [comment]
@mungpara thanks for your PR.
However, I am unable to spot the visual difference from the current release. 
can you please provide a screenshot and description what exactly is supposed to be changed?

**Comment by mungpara** [comment]
> @mungpara thanks for your PR.
> However, I am unable to spot the visual difference from the current release.
> can you please provide a screenshot and description what exactly is supposed to be changed?

look top left and top right border of search box

**Comment by macjohnny** [comment]
this is what I get when I check out your branch:
mungpara-patch-1

this is due to
https://github.com/bithost-gmbh/ngx-mat-select-search/blob/a2ee9ab7a89ea125bbd3341469cf509d5fbf6537/src/app/mat-select-search/mat-select-search.component.scss#L30
if i remove this line, I get:
mungpara-patch-1-improved

File: src/app/mat-select-search/mat-select-search.component.scss [hunk 1]
```diff
@@ -58,6 +59,7 @@ $mat-option-height: 3em;
 }
 
 .mat-select-search-input {
+   border-radius: 4px;
```
Comment (macjohnny) [inline src/app/mat-select-search/mat-select-search.component.scss]: this isn't necessary, as it does not have an additional effect
```suggestion
```

File: src/app/mat-select-search/mat-select-search.component.scss [hunk 2]
```diff
@@ -14,6 +14,7 @@ $mat-option-height: 3em;
   visibility: hidden;
 }
 .mat-select-search-inner {
+  border-radius: 4px 4px 0 0;
```
Comment (macjohnny) [inline src/app/mat-select-search/mat-select-search.component.scss]: remove
https://github.com/bithost-gmbh/ngx-mat-select-search/blob/a2ee9ab7a89ea125bbd3341469cf509d5fbf6537/src/app/mat-select-search/mat-select-search.component.scss#L30

**Comment by macjohnny** [comment]
@mungpara why did you close the PR?

**Pull request closed without merging**